test/nl/surf/eduhub_rio_mapper/smoke_test.clj

```diff
@@ -222,6 +222,22 @@
                   :opleidingeenheidcode "1010O8815"}
                  (:dry-run result))))))
 
+    (testing "education-specifications error real life case"
```

Comment:
What "error"?  The result below seems fine to me.  Please improve this description.